docs_source_files/content/driver_idiosyncrasies/_index.zh-cn.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Driver idiosyncrasies"
+chapter: true
+weight: 10
+---
+
+{{% notice info %}}
+<i class="fas fa-language"></i> 页面需要从英语翻译为简体中文。
+您熟悉英语与简体中文吗？帮助我们翻译它，通过 pull requests 给我们！
+如果您对 GitHub 不熟悉，亦可将翻译内容通过邮箱 <liushilive@outlook.com> 传递给我来提交，感谢您的参与！
```

Comment:
Let's avoid directing people to personal emails, the contribution text should be enough for people to get started. Maybe the contribution doc needs enhancement? 
(The email thing applies to all the other files as well)

Reply:
Is there a more friendly way to get people to submit code.

Many times in China access to GitHub is slow.

Reply:
Using git is the more friendly way to submit code I know. 
I understand that sometimes it can be challenging, but relying on personal emails is not really a way to solve it. 
If something, as mentioned we can just improve the contribution doc.

Reply:
I'll fix this later in the push.